src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -40,8 +39,8 @@ object Lorre extends App with TezosErrors with LazyLogging {
   private val sleepIntervalInSeconds = conf.getInt("lorre.sleepIntervalInSeconds")
   private val feeUpdateInterval = conf.getInt("lorre.feeUpdateInterval")
 
-  lazy val db = DatabaseUtil.db
-  val tezosNodeOperator = new TezosNodeOperator(new TezosNodeInterface())
+  lazy val db = ApiNetworkOperations.getDatabaseMap(network)
```

Comment:
Let's check for both `platform` and `network`. You will have to start asking for `platform` as well from the command line. This might be a good time to use an argument parsing library.

Reply:
I used simple parsing for now, at the moment we have 2 arguments platform and network, in case it will become more complicated then we should think abut library, at the moment I don't think is necessary